data.json

```diff
@@ -1144,19 +1144,24 @@
     "githubUsername": "UtkarshMishra12",
     "imageUrl": "(https://user-images.githubusercontent.com/67385503/126064330-eb1f90d9-d7d3-462c-b032-b381ed00153c.png)",
     "issueNumber": 600
-
   },
   {
     "name": "Rohit Yadav",
     "githubUsername": "rohitturbot",
     "imageUrl": "https://user-images.githubusercontent.com/40739903/126065686-40212b44-01a8-4c5f-b38a-de22ca76cc9c.png",
     "issueNumber": 602
   },
+  {
+
```

Comment:
You can remove this line as it breaks the code style :) Just remove that empty line and the PR is good to go

Reply:
Line no `1155`

Reply:
Ok, I will now.

Reply:
I can't see where to remove line `1155`: 
![image](https://user-images.githubusercontent.com/70807500/126072730-f7fffeb4-4f56-48f7-afcb-e80c1475eb49.png)

Reply:
Just found and did it. Thank you.